webclient/app/src/app/commons/relationshipAccordion/RelationshipAccordion.jsx

```diff
@@ -39,93 +41,105 @@ function RelationshipAccordion(props) {
+                                        regex={RegexConfig.relationship}
+                                    />
+                                </FormControl>
+                            </Grid>
+                            <Grid item sm={1}>
+                                <Typography gutterBottom>{t("relationship.target")}</Typography>
+                            </Grid>
+                            <Grid item sm={5}>
+                                <FormControl fullWidth>
+                                    <InputLabel
+                                        id="targetEntity"
+                                        error={!RegexConfig.entityTitle.test(relationship.otherEntityName)}
+                                    >{t("relationship.targetEntity") + "*"}</InputLabel>
+                                    <Select
+                                        error={!RegexConfig.entityTitle.test(relationship.otherEntityName)}
+                                        labelId="targetEntity"
+                                        id="targetEntitySelect"
+                                        value={relationship.otherEntityName}
+                                        label={t("relationship.targetEntity") + "*" }
+                                        onChange={(event) => editRelationshipProperty("otherEntityName", event.target.value)}
+                                    >
+                                        {targetEntities.map(entity => (
+                                            <MenuItem value={entity.name} key={entity.id} >{entity.name}</MenuItem>
+                                        ))}
+                                    </Select>
+                                </FormControl>
+                                <div className={relationshipStyles.spacerBottom}/>
+                                <FormControl fullWidth>
+                                    <ValidatedTextField
+                                        label={ t("relationship.targetField") }
+                                        value={relationship.otherEntityRelationshipName}
+                                        onChange={(event) => editRelationshipProperty("otherEntityRelationshipName", event.target.value)}
+                                        helperText={ t("relationship.targetField.description", {entityName: currentEntity.name ? currentEntity.name : t("relationship.sourceEntity")} )}
+                                        regex={RegexConfig.relationship}
+                                    />
+                                </FormControl>
+                            </Grid>
+                        </Grid>
+                    </AccordionDetails>        
+                </Accordion>
+            </Grid>
+            <Grid item sm={1}>
+                <IconButton
+                    aria-label="delete"
+                    onClick={handleDelete}
+                >
+                    <Delete />
```

Comment:
Im not quite happy with the current structuring of the delete button. It is certainly good for now but maybe we have to implement this button in the Accordion itself due to Responsive design issues.